src/test/java/org/killbill/billing/plugin/payment/TestingStatesTest.java

```diff
@@ -22,49 +22,49 @@
 
 public class TestingStatesTest {
 
-    TestingStates testingStates = new TestingStates();
+    private final TestingStates testingStates = new TestingStates();
 
     @Test
     void testAddKnownMethod() {
-        Assert.assertEquals(this.testingStates.add(TestingStates.Actions.ACTION_RETURN_PLUGIN_STATUS_CANCELED,
-                                                   "authorizePayment"), true);
+		Assert.assertTrue(this.testingStates.add(TestingStates.Actions.ACTION_RETURN_PLUGIN_STATUS_CANCELED,
```

Comment:
Whitespace seems off, can you reformat in IntelliJ?

Reply:
This looks correct in IntelliJ.